src/app/pages/blog/blog.scss

```diff
@@ -5,6 +5,7 @@
     background-color: ui-color(white);
     justify-items: center;
     width: 100%;
+    overflow: clip;
```

Comment:
This fixes a layout bug I noticed in the blog article: there is a sticky item that shows how much time is left in the article. It was not being sticky.